### ISSUE TITLE
Project: ROWMAN Phase 4

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#422]
Phase 4 of the ROWMAN Replacement Project aims to provide staff and the public with real time permitted road closure data. 

## Background
Construction related road closure permits often give the contractor an approved timeframe for closing the road. For example, ATD may give a contractor 7 calendar days to perform 2 days of work. As such, it is impossible to know when a contractor is actually in the right of way. This hinders ATD’s ability to coordinate activity in the Right of Way, and it impacts the traveling public’s ability to plan their routes. 

Due to the nature of construction, giving the contractor a ‘window’ is necessary, so we must design a solution that allows for us to continue giving them a window and will also allow us to gather data from the contractor on-site, when they are closing and re-opening the Right of Way. 

## Scope & Deliverables
**This project includes:**
- Updating internal permitting workflows to include processes that will make real-time road closure data possible.
- Building a customer-facing solution for permit holders that will allow them to easily notify ATD when they begin a road closure, and when they end it. 
- Designing the real-time road closure data to conform with Work Zone Data Exchange specifications.
- Publishing this data for public use, especially for existing public mapping tools like Google Maps and Waze

## Desired Outcomes
Success for this project means city staff, permit analysts, right of way coordinators, and the public will have access to real-time permitted road closure information. This will reduce the amount of permitting conflicts in the Right-of-Way, which, when they happen, have adverse effects on the contractor’s ability to perform their work and can cause unnecessary traffic congestion. This will also allow the public to better plan their routes and avoid delays caused by permitted road closures. 

We also hope that by openly publishing this data, companies that provide mapping products will be able to integrate this data into their product and provide their users with real-time closure data. 

## Timebox
This is a complex project with heavy scoping/analysis needs. While we have already performed much of the scoping, we will need to refresh the analysis when the project is picked up. 

**Expected Timeline:**

- Scoping & Analysis: 1 Month
- Review Possible Solutions: 1 Month
- Writing Requirements and Acceptance Criteria: 2 Weeks
- Development: 1-2 Months
- Testing: 1 Month
- Training Staff & Permit Holders: 1 Month
- Total Duration From Start to Go-Live: **6 Months**

---
This Github issue represents a project of Austin Transportation's [Data & Technology Services](https://product.austinmobility.io/) team. Project status is documented regularly in the comments below.

**Comment by TracyLinder** [comment]
### Update

This phase is active as of 8/12/19. I'm currently working on a proof of concept that I'll show to the team on 8/23. Once a high level scope of work is approved by Ryan Ward, a request for an estimate will be sent to Unisys. Next, funding will need to be secured from the Case Management Governing Board. If all goes well, development could start in October.

**Comment by johnclary** [comment]
Currently defining all of the acceptance criteria for this project, to be completed by mid-December.

**Comment by TracyLinder** [comment]
**Status Update**
The SOW has been reviewed with stakeholders. This phase is likely to be on hold until the portal replacement is done.

**Comment by TracyLinder** [comment]
### Planning & Prioritizing Meeting Notes

Do we still want to develop this?
- Yes

Priority?
- Still a high priority from the audit

Misc Notes
- CPO wants to be involved in piloting, testing, design, perhaps funding if needed (G. Pep)
- Ryan mentioned looking into DotMap as part of the solution
- We do want to scope alternative solutions (right now the plan is to use portal as a solution)
